lib/rubocop/cop/performance/double_start_end_with.rb

```diff
@@ -42,9 +42,7 @@ def on_or(node)
                 receiver: receiver.source,
                 method: method,
                 combined_args: combine_args(first_call_args, second_call_args),
-                original_code: node.source
```

Comment:
Seems there's a bug in the cop. When there's no element on the first line, the current indentation seems to me to be the preferable one.